### PR TITLE
[FLINK-13979] [docs-zh] Translate new streamfilesink docs to chinese

### PR DESCRIPTION
**Pull request by catkint**
## What is the purpose of the change

Translate new streamfilesink docs to chinese

## Brief change log
Translate new streamfilesink docs to chinese

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`:  no
  - The serializers: no
  - The runtime per-record code paths (performance sensitive):no
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: no
  - The S3 file system connector: no

## Documentation

  - Does this pull request introduce a new feature? no

### CODE REVIEW

File: docs/dev/connectors/streamfile_sink.zh.md [hunk 1]
```diff
@@ -28,80 +28,69 @@ under the License.
...
+ - [DateTimeBucketAssigner]({{ site.javadocs_baseurl }}/api/java/org/apache/flink/streaming/api/functions/sink/filesystem/bucketassigners/DateTimeBucketAssigner.html) ：默认基于时间的分配器
+ - [BasePathBucketAssigner]({{ site.javadocs_baseurl }}/api/java/org/apache/flink/streaming/api/functions/sink/filesystem/bucketassigners/BasePathBucketAssigner.html) ：将所有部分文件（part file）存储在基本路径中的分配器（单个全局桶）
 
-### Rolling Policy
+### 滚动策略
 
-The [RollingPolicy]({{ site.javadocs_baseurl }}/api/java/org/apache/flink/streaming/api/functions/sink/filesystem/RollingPolicy.html) defines when a given in-progress part file will be closed and moved to the pending and later to finished state.
-In combination with the checkpointing interval (pending files become finished on the next checkpoint) this controls how quickly
-part files become available for downstream readers and also the size and number of these parts.
+滚动策略 [RollingPolicy]({{ site.javadocs_baseurl }}/api/java/org/apache/flink/streaming/api/functions/sink/filesystem/RollingPolicy.html) 定义了指定的文件在何时关闭（closed）并将其变为 Pending 状态，随后变为 Finished 状态。与 Checkpoint 间隔（处于 Pending 状态的文件会在下一次 Checkpoint 变为 Finished 状态）结合使用，可以控制部分文件（part file）对下游读取者可用的速度以及大小和数量。
```
Comment (xintongsong) [inline docs/dev/connectors/streamfile_sink.zh.md]: "与 Checkpoint 间隔（处于 Pending 状态的文件会在下一次 Checkpoint 变为 Finished 状态）结合使用，可以控制部分文件（part file）对下游读取者可用的速度以及大小和数量。"
这句话尽管翻译是准确的，但如果不看英文原文只看中文会比较难以理解，建议重新组织一下语言。

File: docs/dev/connectors/streamfile_sink.zh.md [hunk 2]
```diff
@@ -28,80 +28,69 @@ under the License.
...
 
-File naming schemes:
- - **In-progress / Pending**: `part-subtaskIndex-partFileIndex.inprogress.uid`
- - **Finished:** `part-subtaskIndex-partFileIndex`
+文件命名方案：
+ - **In-progress / Pending**：`part-subtaskIndex-partFileIndex.inprogress.uid`
+ - **Finished** ：`part-subtaskIndex-partFileIndex`
 
-Part file indexes are strictly increasing for any given subtask (in the order they were created). However these indexes are not always sequential. When the job restarts, the next part index for all subtask will be the `max part index + 1`.
+对于任何给定的 Subtask （按照它们的创建顺序），部分文件（part file）的索引严格递增。但是，这些索引并不总是连续的。当作业重启时，所有 Subtask 部分文件（part file）索引的值将是 `max part index + 1` 。
```
Comment (xintongsong) [inline docs/dev/connectors/streamfile_sink.zh.md]: ”（按照它们的创建顺序）“这个应该是part file的创建顺序，而非subtask的创建顺序

File: docs/dev/connectors/streamfile_sink.zh.md [hunk 3]
```diff
@@ -28,80 +28,69 @@ under the License.
...
 
-Each writer subtask will have a single in-progress part file at any given time for every active bucket, but there can be several pending and finished files.
+对于每个活动的桶，写入 Subtask 在任何时候都只有一个处于 In-progress 状态的部分文件（part file），但是可能有几个 Penging 和 Finished 状态的部分文件（part file）。
```
Comment (xintongsong) [inline docs/dev/connectors/streamfile_sink.zh.md]: ”写入Subtask” 有歧义，只看中文会产生到底是subtask向其他地方（部分文件）写数据，还是从别的地方向subtask写数据这样的疑惑。这里subtask应该是writer

**Comment by catkint** [comment]
I will modify it asap. thx.

**Pull request closed without merging**